### PR TITLE
Added script for one click start

### PR DESCRIPTION
**Pull request by Devaniti**

### CODE REVIEW

**Review by Arriven — APPROVED** [review]
Looks good to me, but can you move it into a scripts/ folder please? I'll merge in couple minutes anyway but having better file structure would be nice considering the amount of controbutors

File: scripts/StartDB1000N.bat [hunk 1]
```diff
@@ -0,0 +1,28 @@
+echo $SaveFolder = $args[0]                                                       >  %temp%\GetDB1000N.ps1
```
Comment (Amet13) [inline scripts/StartDB1000N.bat:1]: @Devaniti is there any option to add a short comment in this script, what does it for? For people who are not familiar with Windows?

Comment (Devaniti) [reply scripts/StartDB1000N.bat:1]: makes sense

Comment (Arriven) [reply scripts/StartDB1000N.bat:1]: Let's do it as a separate PR

**Pull request merged**